authority-portal-backend/docs/dev/keycloak-realm-export/README.md

```diff
@@ -13,6 +13,15 @@ To run this script, execute the following command from the backend project's roo
 ./docs/dev/keycloak-realm-export/extract-realm.sh
 ```
 
+After the script has run, the exported [realm.json](../../../authority-portal-quarkus/src/main/resources/realm.json) may contain unrecognized fields which leads to an import error on next startup.
+Quarkus indicates those with an error message on startup like this:
```

Comment:
I suppose this issue is fixed by the newest version or cheap workarounds depending on the exact property causing an error: https://stackoverflow.com/questions/78575033/quarkus-and-keycloak-compatibility

The question is, how long will it take to troubleshoot it and do we need to do this?